### PR TITLE
Remove PrestaShop Translator Adapter and use Symfony default one

### PR DESCRIPTION
**Pull request by julienbourdeau**
| Questions | Answers |
| --- | --- |
| Branch? | develop |
| Description? | At the time we introduce a prestashop translator (implementing the SymfonyInterface). This translator have been fully remove to use the symfony default one. |
| Type? | refacto |
| Category? | CO |
| BC breaks? | no |
| Deprecations? | no |
| Fixed ticket? |  |
| How to test? | Test trads when it's ready |

### CODE REVIEW

File: controllers/front/.codemod.bookmark [hunk 1]
```diff
@@ -0,0 +1 @@
+./AddressController.php:131
```
Comment (mickaelandrieu) [inline controllers/front/.codemod.bookmark]: hmm ?

**Comment by mickaelandrieu** [comment]
Wow, just "wow". Great improvement :)

File: controllers/front/.codemod.bookmark (hunk 1 above)
Comment (julienbourdeau) [inline controllers/front/.codemod.bookmark]: 😩

File: controllers/front/BestSalesController.php [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <?php
 /**
- * 2007-2015 PrestaShop
+ * 2007-2015 PrestaShop.
```
Comment (Shine-neko) [inline controllers/front/BestSalesController.php]: 2016 ? 💃

Comment (mickaelandrieu) [inline controllers/front/BestSalesController.php]: later, let's focus on the pull request please

**Comment by mickaelandrieu** [comment]
@julienbourdeau are the tests working on your own computer ?

**Comment by julienbourdeau** [comment]
@mickaelandrieu I think it's because of module that I updated locally but, of course, travis is not aware of it. I'll update the composer.json temporarily.

**Pull request merged**